### ISSUE TITLE
Bug: "text" too greedy

### ISSUE THREAD

**Issue by OptimisticLock** [wanasit/chrono#99]
Awesome package, thank you!

`chrono.parse("In 5 minutes I will go home")[0].text `evaluates to `
"In 5 minutes I"`

Why the I?

Also, why does not chrono pick "in a year"?

**Comment by wanasit** [comment]
Thanks for the bug report. It was mis-recognizing "I" as a timezone abbr.
```
> new Date("Sat Mar 12 2016 14:48:11 (JST)") 
// Sat Mar 12 2016 14:48:11 GMT+0900 (JST)
> new Date("Sat Mar 12 2016 14:48:11 (I)")
// Sat Mar 12 2016 14:48:11 GMT+0900 (JST)
```

Fixed it in 899d59d. Please pull ver 1.1.4

**Comment by OptimisticLock** [comment]
Thank you
